src/App.css

```diff
@@ -29,9 +29,9 @@ button {
   font-weight: 600;
 }
 
-input[type="text"] {
+.display {
   height: 40px;
-  width: 300px;
+  width: 304px;
   background-color: grey;
   border: 0;
   color: #fff;
```

Comment:
- **[OPTIONAL]** We are going to work on the styles in another milestone. I recommend using 👉🏽  `display: flex;` and `justify-content: flex-end;` 👈🏽 to match the original design.